core/serialization/download_json_serializer.py

```diff
@@ -104,7 +104,6 @@ def __init__(self, programme_progress: entities.ProgrammeProgress):
 
     def to_json(self):
         return {
-            "ID": str(self.programme_progress.id),
```

Comment:
the version of data model that I have still has these ID's on, but I presume mine must be out of date? Since it doesn't really make sense to me to include them

Reply:
Maeve has told me they can go